Framework/Kernel/test/ArrayPropertyTest.h

```diff
@@ -68,9 +68,9 @@ class ArrayPropertyTest : public CxxTest::TestSuite {
 
     // Test vector of vector.
     std::vector<std::vector<int>> input;
-    input.push_back(std::vector<int>(10)); // Make it 10 elements long, but
-                                           // should only be the size of the
-                                           // parent vector that is counted.
+    input.emplace_back(10); // Make it 10 elements long, but
+                            // should only be the size of the
+                            // parent vector that is counted.
```

Comment:
One could argue that the new syntax is a bit unclear. But I have no strong opinion.